host: 192.168.10.100
mounting_type: downward
add_intensity: disadd

